test/math/FloatMatrixTest.ooc

```diff
@@ -13,127 +14,126 @@ FloatMatrixTest: class extends Fixture {
-			checkAllElements(a + b, [10.0f, 10.0f, 10.0f, 10.0f, 10.0f, 10.0f, 10.0f, 10.0f, 10.0f])
-			checkAllElements(a + b give() + a give(), [11.0f, 12.0f, 13.0f, 14.0f, 15.0f, 16.0f, 17.0f, 18.0f, 19.0f])
+			a := this createMatrix(3, 3, [1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f]) take()
+			b := this createMatrix(3, 3, [9.0f, 8.0f, 7.0f, 6.0f, 5.0f, 4.0f, 3.0f, 2.0f, 1.0f]) take()
+			this checkAllElements(a + b, [10.0f, 10.0f, 10.0f, 10.0f, 10.0f, 10.0f, 10.0f, 10.0f, 10.0f])
+			this checkAllElements(a + b give() + a give(), [11.0f, 12.0f, 13.0f, 14.0f, 15.0f, 16.0f, 17.0f, 18.0f, 19.0f])
 		})
 
 		this add("subtraction", func {
 			a := createMatrix(3, 3, [1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f]) take()
 			b := createMatrix(3, 3, [9.0f, 8.0f, 7.0f, 6.0f, 5.0f, 4.0f, 3.0f, 2.0f, 1.0f])
-			checkAllElements(a - b, [-8.0f, -6.0f, -4.0f, -2.0f, 0.0f, 2.0f, 4.0f, 6.0f, 8.0f])
-			checkAllElements(a - a give(), [0, 0, 0, 0, 0, 0, 0, 0, 0])
+			this checkAllElements(a - b, [-8.0f, -6.0f, -4.0f, -2.0f, 0.0f, 2.0f, 4.0f, 6.0f, 8.0f])
+			this checkAllElements(a - a give(), [0, 0, 0, 0, 0, 0, 0, 0, 0])
 		})
 
 		this add("solver (square)", func {
 			// Solve a * x = y
-			a := createMatrix(5, 5, [1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 1.0f, 3.0f, 6.0f, 10.0f, 15.0f, 1.0f, 4.0f, 10.0f, 20.0f, 35.0f, 1.0f, 5.0f, 15.0f, 35.0f, 70.0f])
-			y := createMatrix(1, 5, [-1.0f, 2.0f, -3.0f, 4.0f, 5.0f])
+			a := this createMatrix(5, 5, [1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 1.0f, 3.0f, 6.0f, 10.0f, 15.0f, 1.0f, 4.0f, 10.0f, 20.0f, 35.0f, 1.0f, 5.0f, 15.0f, 35.0f, 70.0f])
+			y := this createMatrix(1, 5, [-1.0f, 2.0f, -3.0f, 4.0f, 5.0f])
 			x := a solve(y)
-			checkAllElements(x, [-70.0f, 231.0f, -296.0f, 172.0f, -38.0f])
+			this checkAllElements(x, [-70.0f, 231.0f, -296.0f, 172.0f, -38.0f])
 		})
 
 		this add("solver (non-square)", func {
-			a := createMatrix(2, 3, [2.0f, 0.0f, 4.0f, 2.0f, 4.0f, 6.0f])
-			y := createMatrix(1, 3, [1.0f, -2.0f, 1.0f])
+			a := this createMatrix(2, 3, [2.0f, 0.0f, 4.0f, 2.0f, 4.0f, 6.0f])
+			y := this createMatrix(1, 3, [1.0f, -2.0f, 1.0f])
 			x := a solve(y)
-			checkAllElements(x, [1.0f, -0.5f])
+			this checkAllElements(x, [1.0f, -0.5f])
 		})
 
 		this add("set and get", func {
-			matrix = createMatrix(3, 3, [1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f])
-			matrix[0, 0] = 42.0f
-			expect(matrix[0, 0], is equal to(42.0f) within(this precision))
+			m := this createMatrix(3, 3, [1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f])
+			m take()[0, 0] = 42.0f
+			expect(m[0, 0], is equal to(42.0f) within(this precision))
 		})
 
 		this add("print columns", func {
-			A := createMatrix(3, 3, [1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f]) take()
-			column := A getColumn(1)
+			a := this createMatrix(3, 3, [1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f]) take()
+			column := a take() getColumn(1)
```

Comment:
Doesn't make a difference here, but just so I understand, you don't need to `take()` here since you did it on the above line.. right?